packages/components/.gitignore

```diff
@@ -1,32 +1,9 @@
 # See https://help.github.com/ignore-files/ for more about ignoring files.
 
-# dependencies
-/node_modules
-
-# testing
-/coverage
-
 # production
-/build
-/dist
 /css
```

Comment:
You should be able to ignore `css` at the root since we tend to not write CSS directly (we use scss) or include any folders called `css`

Reply:
<img width="222" alt="image" src="https://github.com/deephaven/web-client-ui/assets/55671206/5aeafedf-d265-459d-a763-6341f037a837">
do these two lines accomplish the same thing? If so I assume I would just keep the 2nd one since it's shorter

Reply:
No they don't. Although the 1st won't do anything in the root of our current repo because we don't have a `src` folder in the root. So you can remove the 1st

The first is any file ending in `.css` inside the `src` folder relative to the `.gitignore` location

The second is any folder called `css` at any depth.

The only place we have a tracked CSS file is `packages/icons/docs/style.css`. Everywhere else is `.scss` or generated `css` that we don't track because it's a build artifact.